keyboards/sol/common/i2c_master.c

```diff
@@ -0,0 +1,223 @@
+/* Library made by: g4lvanix
```

Comment:
This file should NOT be copied here.   

There is a copy of i2c_master.*  in /drivers/(avr/arm)/, and that is what should be used. 
If that doesn't work, it should be updated to work with your code, while ensuring compatibility with existing code.  
At least, if possible.   

That way, we avoid the code fragmentation issue that we see with various aspects of the code, already.

Reply:
makes sense, is that directory part of the standard includes search path?

Reply:
fixed